### DISCUSSION TITLE
WalletConnect 2.0 + Web3Modal support for Solana?

### DISCUSSION THREAD

**Discussion by olivertang** [WalletConnect/walletconnect-monorepo#1719] [Developer Support]
Is there an ETA on the Solana integration effort for web3modal & WalletConnect 2.0? Our team needs to support login via WalletConnect for customers using Fireblocks on Solana. We saw that Web3Modal currently does not support this yet - however, we saw a PR in draft mode & wasn't sure if this is still being worked on: 

PR: 
https://github.com/WalletConnect/web3modal/issues/679

**Comment by olivertang** [comment]
Following up here - got an answer for v2.1, early next year 

https://github.com/WalletConnect/web3modal/pull/653#issuecomment-1338911233

**Comment by xzilja** [comment]
It is more manual, but you can also use `@web3modal/standalone` together with `@walletconnect/sign-client` for this already. https://docs.walletconnect.com/2.0/web3modal/standalone/installation

But until `2.1` you'll have to manage a lot of stuff liek injected connectors.
